specification/search/data-plane/Azure.Search/preview/2020-06-30/searchservice.json

```diff
@@ -5711,6 +5711,14 @@
           "x-ms-client-name": "ETag",
           "type": "string",
           "description": "The ETag of the data source."
+        },
+        "encryptionKey": {
+          "$ref": "#/definitions/SearchResourceEncryptionKey",
+          "description": "A description of an encryption key that you create in Azure Key Vault. This key is used to provide an additional level of encryption-at-rest for your datasource definition when you want full assurance that no one, not even Microsoft, can decrypt your data source definition in Azure Cognitive Search. Once you have encrypted your data source definition, it will always remain encrypted. Azure Cognitive Search will ignore attempts to set this property to null. You can change this property as needed if you want to rotate your encryption key; Your datasource definition will be unaffected. Encryption with customer-managed keys is not available for free search services, and is only available for paid services created on or after January 1, 2019.",
+          "externalDocs": {
+            "url": "https://aka.ms/azure-search-encryption-with-cmk"
+          },
+          "x-nullable": true
```

Comment:
Could you just not serialize it instead sending an explicit `null`?

Reply:
This is what we have been doing since all our old API versions - we could have set the json serialization to ignore on null in responses, but we don't...

Btw, this is what we do for other instances of the `encryptionKey` property in other metadata like indexes/synonym maps